scripts/pylint_extensions.py

```diff
@@ -37,7 +37,7 @@
 EXCLUDED_PHRASES = [
     'utf', 'pylint:', 'http://', 'https://', 'scripts/', 'extract_node']
 
-DATA_TYPES = ['int', 'str', 'float', 'bool']
+DATA_TYPES = ['int.', 'str.', 'float.', 'bool.']
```

Comment:
1. The period is not, conceptually, part of the data type, so it should not be here. Add it only when you are doing the regex check.

2. Are you using DATA_TYPES in just a single part of the codebase? If so, consider making this a class-level constant perhaps, to reduce its visibility accordingly.

Reply:
Done.

Reply:
Done.